proc/exec_darwin.c

```diff
@@ -97,6 +97,8 @@ fork_exec(char *argv0, char **argv, int size,
 		exit(1);
 	}
 
+	sleep(1);
```

Comment:
lldb does it but provides no explanation. I suspect it is just to avoid introducing race conditions accidentally, rather than circumventing one, but I don't know.